engine/src/main/java/org/terasology/engine/rendering/primitives/ChunkMesh.java

```diff
@@ -223,33 +207,28 @@ public int render(RenderPhase type) {
      * ChunkMesh instances cannot be un-disposed.
      */
     public void dispose() {
-        lock.lock();
-        try {
-            if (!disposed) {
-                for (int i = 0; i < vertexBuffers.length; i++) {
-                    int id = vertexBuffers[i];
-                    if (id != 0) {
-                        GL15.glDeleteBuffers(id);
-                        vertexBuffers[i] = 0;
-                    }
-
-                    id = idxBuffers[i];
-                    if (id != 0) {
-                        GL15.glDeleteBuffers(id);
-                        idxBuffers[i] = 0;
-                    }
-
-                    id = vaoCount[i];
-                    if (id != 0) {
-                        GL30.glDeleteVertexArrays(id);
-                        vaoCount[i] = 0;
-                    }
+        if (!disposed) {
+            for (int i = 0; i < vertexBuffers.length; i++) {
+                int id = vertexBuffers[i];
+                if (id != 0) {
+                    GL15.glDeleteBuffers(id);
```

Comment:
Why use `GL15` but change other occurrences to `GL30`?

Reply:
they end up mapping to the same call.